packages/pds/src/services/moderation/views.ts

```diff
@@ -605,7 +606,10 @@ export class ModerationViews {
   }
 }
 
-type RepoResult = DidHandle & RepoRoot
+type RepoResult = DidHandle &
+  RepoRoot & {
+    emailConfirmedAt?: string | null
```

Comment:
Is there a reason this field should be optional?  My concern is that it could create inconsistencies in the resulting views, where some responses contain `emailConfirmedAt` and others don't.

Reply:
clarified in the [above comment](https://github.com/bluesky-social/atproto/pull/1757#discussion_r1376151206)